### ISSUE TITLE
[a11y] SubPlat payment page missing text labels

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#11369]
On the SubPlat payment/checkout page, there are multiple elements without text labels. This makes it difficult to navigate with a screenreader, and is not WCAG compliant.

elements:
2 document elements, name is null, DOMNode: #document
4 diagram elements, name is null, DOMNode: svg
3 credit card elements, role is "entry"

* Credit or debit card number
* Credit or debit card expiration date
* Credit or debit card CVC/CVV

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXA-4339)

**Comment by data-sync-user** [comment]
➤ Anna Sobiepanek commented:

Is there an alternative to using the iframe?if not lets cancel
